tests/rest/test_single_topology_rest.py

```diff
@@ -111,11 +113,13 @@ def test_single_topology_rest_vacuum(mol_pair, temperature_scale_interpolation_f
     else:
         assert energy_scale < 1.0
 
+        # check that a proper subset of ligand-ligand nonbonded interactions are scaled
+        assert not np.isclose(U_nonbonded, U_nonbonded_ref)
+        assert not np.isclose(U_nonbonded, energy_scale * U_nonbonded_ref)
+
         if has_rotatable_bonds or has_aliphatic_rings:
             assert 0 < len(st_rest.candidate_propers)
-
-            if energy_scale < 1.0:
-                assert U_proper < U_proper_ref
+            assert U_proper < U_proper_ref
```

Comment:
Should also sanity check that this isn't just testing some small floating point round-off, eg. `0.99999<1.00000`

Reply:
Added assertion in https://github.com/proteneer/timemachine/pull/1535/commits/ef9abac03369cc1270759ad4590f0e209050b493